tests/backends/cursors/sqlite.py

```diff
@@ -0,0 +1,38 @@
+
+from django.core.exceptions import ImproperlyConfigured
+
+try:
+    from django.db.backends.sqlite3.base import SQLiteCursorWrapper
+    HAS_SQLITE = True
+except ImproperlyConfigured as exc:
+    HAS_SQLITE = False
+
+if HAS_SQLITE:
```

Comment:
Please change to @unittest.skipUnless(connection.vendor == 'sqlite', ... instead of try/except.